src/Elastic.Apm/Model/SpanTypeAndSubtype.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to Elasticsearch B.V under
+// one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+namespace Elastic.Apm.Model
+{
+	/// <summary>
+	/// Encapsulates type and subtype
+	/// </summary>
+	internal struct SpanTypeAndSubtype
```

Comment:
minor nitpick: naming

```suggestion
	internal struct SpanTimerKey
```

Reply:
Done.